src/dotnet-scaffolding/dotnet-scaffold/Telemetry/CommandExecuteTelemetryEvent.cs

```diff
@@ -0,0 +1,21 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+using Microsoft.DotNet.Scaffolding.Core.ComponentModel;
+using Microsoft.DotNet.Scaffolding.Internal.Telemetry;
+
+namespace Microsoft.DotNet.Tools.Scaffold.Telemetry;
+
+internal class CommandExecuteTelemetryEvent : TelemetryEventBase
+{
+    private const string TelemetryEventName = "CommandExecute";
+    public CommandExecuteTelemetryEvent(DotNetToolInfo dotnetToolInfo, CommandInfo commandInfo, int? exitCode) : base(TelemetryEventName)
+    {
+        SetProperty("PackageName", dotnetToolInfo.PackageName);
+        SetProperty("Version", dotnetToolInfo.Version);
+        SetProperty("ToolName", dotnetToolInfo.Command);
```

Comment:
So many of these things (ToolName, CommandName, CommandCategory) become potentially PII when we allow extensibility. We'll probably need to talk to the privacy review folks about the right thing to do (always hash and have a well-known hash list? Flag first-party and only hash non-first party?).

Reply:
should I leave these in rn and we can address this in a privacy review?

Reply:
Leave it for now - let's try to talk to Phil tomorrow at least. I'd be hesitant to ship even a preview version wirhout the right answer because once the telemetry gets categorized on the backend it's very hard to change.